content/projects/data-validation-with-python-and-regex/_index.md

```diff
@@ -1,3 +1,9 @@
 ---
-title: data-validation-with-python-and-regex
----
\ No newline at end of file
+title: data validation with Python and regex
+ready: true
+---
```

Comment:
PLEASE PLEASE PLEASE DRY DRY DRY
please
for realsies

This project exists already. Just update the one that exists. I promise this exists.  It's entitled "string calculator" maybe it needs a better tile.